ros-build-essential/ros-build-essential.spec

```diff
@@ -0,0 +1,25 @@
+Name: ros-build-essential
+Version: 1.0.0
+Summary: ROS Infrastructure variant including packages that are expected to be present when building any ROS packages.
+Release: %{?dist}%{?release_suffix}
```

Comment:
I haven't included a "package revision" integer which we do in the bloom templates to match the debian package version increment but I don't know how necessary it is to have one.

Since this package essentially _is_ the spec file I think any changes to it will warrant a patch release of the version above.

Reply:
Eh, it wouldn't be a bad idea to add one just in case. It does no harm if we don't use it.

Reply:
Added in [ca8ef84](https://github.com/ros-infrastructure/infra-variants/pull/1/commits/ca8ef8436032c5a8ae4d21661bbec77f9a44c948)